### DISCUSSION TITLE
sqlite-file very large [Bug]

### DISCUSSION THREAD

**Discussion by 20pyro00** [Chia-Network/chia-blockchain#11049] [Support]
### What happened?

I am running chia on a Raspberry Pi 4 using linux. I got a message, that my farm stopped working, so i checked on it. There was a JavaScript error, telling me "no space left on device". I then checked the filemanager and saw, that the .sqlite-file is 90GiB!!! My Wallet-file is also 9GiB. Is that normal? I had a lot of problems with my setup and had to restart/reinstall often. Could that somehow broke something?

### Version

1.3.3

### What platform are you using?

Pi4

### What ui mode are you using?

GUI

**Comment by sergree** [comment]
@20pyro00 
Hey!
Yep. I have the same file sizes.
But we can now update database to v2, this will significantly reduce its size:
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-new-database

Use this CLI command to do this:
`chia db upgrade`

**Comment by 20pyro00** [comment]
Hi! Thanks for your reply. I upgraded to V2 and am now resyncing my node. Because i had no free space left, i have to do a full resync. I started the resync yesterday and it is now at ~20% and the .sqlite-file is only 3,5Gb 😃

**Comment by 20pyro00** [comment]
I checked again, and it is still syncing. It slowed down very hard. Then i looked at the filesize: AGAIN 83GB!! What the hell is this?

**Reply by DonMarmueta** [reply]
chia-database to download .db v2 ....

**Reply by DonMarmueta** [reply]
https://chia-database.com/

**Reply by 20pyro00** [reply]
Thanks for your help. I downloaded the file from https://chia.sponix.net/ . It is now syncing the rest, but again at 56Gb. Maybe i need to do a complete reinstall of chia.

**Comment by sergree** [comment]
Idk, everything is fine for me after v2 updating
P.S. I'm not chia dev :)

**Comment by pyroguysf** [comment]
I'm experiencing the same bug, as have a few others I've heard from. I've upgraded the db files using 1.3.1 of both a Windows 10 and RPi4 environment that have been up since July and only saw a ~5% reduction in both cases when the v2 file was finished (87.3 GB v1 to 82.9 GB v2 on Windows). I've also done multiple clean installs of Windows 10 in both VMs and on bare hardware, updated it, installed the latest version of Chia (used 1.3.1 and 1.3.3), put in my keys, turned off UPnP, and let it sync. In every single case, my v2 db files are currently at 90+ GB.

**Reply by 20pyro00** [reply]
Ok, that´s really weird. My farmer finally finished syncing and the sqlite-file is at 57,6Gb. I downloaded the V2 file from https://chia.sponix.net/ and let my machine sync the rest. This worked for me. Maybe it will for you. Give it a try.
